switchboard.py

```diff
@@ -11,6 +12,8 @@ def __init__(self,):
         """Return a new Switchboard instance."""
         self.input_channels = {}
         self.output_channels = {}
+        log.info('Canarydrops switchboard started')
```

Comment:
"Canarytokens" (Canarydrops is an internal legacy term for a particular data structure)